fec.config.js

```diff
@@ -26,15 +27,23 @@ module.exports = {
     }),
   },
   moduleFederation: {
-    moduleName: 'widgetLayout',
+    moduleName: process.env.CONFIG_PORT || process.env.PROXY ? 'frontendStarterApp' : 'widgetLayout', // overwriting starter app locally for ease of development
```

Comment:
No need to add these as we should be using `widgetLayout` by default in landing page.
```suggestion
    moduleName: 'widgetLayout'
```